tensorflow/lite/micro/tools/Debugging_Tool_README.md

```diff
@@ -0,0 +1,47 @@
+# C++ Layer by Layer Output Tool on TFLite Micro
+
+This tool allows you to pass in a TfLite model and returns a flatbuffer file with input and the corresponding output values appended into it that can be passed into a python debugging tool which can compare those golden values vs the x86 TFLM reference kernel implementation.
```

Comment:
I would suggest re-writing this to first describe the problem this tool solves. Why would someone need to use this tool? Then teach them how to use it. The design details aren't particularly important.

There's nominally two workflows that are supported, we should describe each and when to use them.

Reply:
done ! along with other comments